contrib/git/git-hook-scripts/README.md

```diff
@@ -0,0 +1,31 @@
+# git hook scripts
+The subfolders contain common git hook scripts that were updated to be used directly with Gitea. If you are looking for scripts, you can have a look at [Github platform samples](https://github.com/github/platform-samples). If you have own scripts which are from general interest, feel free to share them here.
+
+### local/user scripts
+Those scripts can either be used as local scripts the user can add to his repository (if not disabled in Gitea configuration file).
+
+### global scripts
+Those scripts can also appied to all newly created repositories by git automatically, when following these steps below.
```

Comment:
@"mark" I'm not sure if global scripts are took into account, because gitea create a own env in witch `git` command is running

Reply:
@6543 I had the same thoughts when I first heard about it, but I ran some tests. As long as the templates are assigned to the git user of the Gitea environment (in my case 'git'), those scripts are pulled in automatically once a new reposity is created. AFAIK it is done when `git init` is executed.

Do you suggest canging the text to be more clear about this like
```
Note:
Gitea runs the git command as the (hopefully non-priviledged) user `RUN_USER` defined in the configuration file. 
It is assumed a home directory was created for that user. In oder to execute the following shell commands, replace `<RUN_USER>` with the corresponding user name.
```

I would change the file depending on the outcome of our discussion.

Reply:
@6543 I have made the changes. Can you please have a look at it?